docs/install.md

```diff
@@ -31,7 +31,7 @@ There are several methods to grant the driver IAM permissions:
 * Using IAM [instance profile](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_use_switch-role-ec2_instance-profiles.html) - attach the policy to the instance profile IAM role and turn on access to [instance metadata](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-metadata.html) for the instance(s) on which the driver Deployment will run
 * EKS only: Using [IAM roles for ServiceAccounts](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html) - create an IAM role, attach the policy to it, then follow the IRSA documentation to associate the IAM role with the driver Deployment service account, which if you are installing via Helm is determined by value `controller.serviceAccount.name`, `ebs-csi-controller-sa` by default. If you are using k8s 1.24 or higher, the ServiceAccountToken is not mounted because the `LegacyServiceAccountTokenNoAutoGeneration` feature gate is enabled.
 Therefore, if you are using k8s 1.24 or higher, you need to set `true` to `controller.serviceAccount.autoMountServiceAccountToken`.
-* Using secret object - create an IAM user, attach the policy to it, then create a generic secret called `aws-secret` in the `kube-system` namespace with the user's credentials
+* Using secret object - create an IAM user, attach the policy to it, then create a generic secret in the `kube-system` namespace with the user's credentials. The snippet below creates the generic secret named `aws-secret` that the driver accepts by default. You can customize this default name via Helm by setting `Value.awsAccessSecret.name` to a custom secret name. Additionally, you may use an existing secret with an AWS access key by configuring `Value.awsAccessSecret.name` with the name of your existing secret, `Value.awsAccessSecret.keyId` with the key of the AWS_ACCESS_KEY_ID, and `Value.awsAccessSecret.accessKey` with the key of the AWS_SECRET_ACCESS_KEY. 
```

Comment:
configuring `Value.awsAccessSecret.name` -> configuring the Helm parameter `awsAccessSecret.name`

`Value.awsAccessSecret.keyId` -> `awsAccessSecret.keyId`

`Value.awsAccessSecret.accessKey` -> `awsAccessSecret.accessKey`